packages/apollo-engine-reporting/src/schemaReporter.ts

```diff
@@ -0,0 +1,160 @@
+// Fields required for the trial related emails
+import gql from 'graphql-tag';
+import {
+  AutoregReportServerInfo,
+  ReportServerInfoVariables,
+  EdgeServerInfo,
+} from './reportingOperationTypes';
+import { DocumentNode, execute, makePromise, Observable } from 'apollo-link';
+import { FetchResult } from 'apollo-link/lib/types';
+import { HttpLink } from 'apollo-link-http';
+import { fetch } from 'apollo-server-env';
+import { Logger } from 'apollo-server-types';
+
+const reportServerInfoGql = gql`
+  mutation ReportServerInfo($info: EdgeServerInfo!, $executableSchema: String) {
+    me {
+      __typename
+      ... on ServiceMutation {
```

Comment:
If we are considering renaming ServiceMutation in our API we should consider doing that before this is released? Or at least a hacky initial version of it somehow? cc @pcarrier

Reply:
This is being released with the __experimental namespace, I would prefer we not block on more changes given that this will change again in a few weeks for "hard launch".